packages/components/select/__tests__/select.test.ts

```diff
@@ -1963,21 +1963,21 @@ describe('Select', () => {
     })
     await nextTick(nextTick)
     const wrapperEl = wrapper.find('input').element as HTMLDivElement
-    expect(wrapperEl.style.height).toEqual('40px')
+    expect(wrapperEl.style.height).toEqual('38px')
```

Comment:
I think we can test `.el-input__wrapper` directyly, instead of changing sizes.

Reply:
I tried it. It's `style.htight` an empty string.

Reply:
OK. I think we can write it as a loop by `for`.

Reply:
OK, leave it to me.

Reply:
You are so awesome that I took it straight to the test.